stackable-base/Dockerfile

```diff
@@ -65,6 +65,8 @@ SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
 # https://github.com/stackabletech/docker-images/pull/533
 COPY stackable-base/stackable/dnf.conf /etc/dnf/dnf.conf
 
+# Default curl configuration to avoid forgetting settings and to declutter the Dockerfiles
+COPY stackable-base/stackable/curlrc /root/.curlrc
```

Comment:
Will we also need it for whatever user we switch to (I haven't checked if we do switch users and then curl stuff)?

Reply:
As per future comment, this was already considered and done.